earthfile2llb/interpreter.go

```diff
@@ -416,9 +420,16 @@ func (i *Interpreter) handleCopy(ctx context.Context, cmd spec.Command) error {
 			dest += string(filepath.Separator)
 		}
 		for _, src := range srcs {
-			err = i.converter.CopyArtifact(ctx, src, dest, platform, buildArgs.Args, *isDirCopy, *keepTs, *keepOwn, *chown, *ifExists)
-			if err != nil {
-				return WrapError(err, cmd.SourceLocation, "copy artifact")
+			if i.local {
+				err = i.converter.CopyArtifactLocal(ctx, src, dest, platform, buildArgs.Args, *isDirCopy)
+				if err != nil {
+					return WrapError(err, cmd.SourceLocation, "copy artifact locally")
+				}
+			} else {
+				err = i.converter.CopyArtifact(ctx, src, dest, platform, buildArgs.Args, *isDirCopy, *keepTs, *keepOwn, *chown, *ifExists)
+				if err != nil {
+					return WrapError(err, cmd.SourceLocation, "copy artifact")
+				}
 			}
 		}
 	} else {
```

Comment:
This else case should be handled too somehow. Maybe not in this PR, but at the very least we should return an error when attempting

```
LOCALLY
COPY ./some-file ./
```

Reply:
that's a good point, I overlooked this; i'll change it to an error for this PR and replicate+fix later